tests/conftest.py

```diff
@@ -22,3 +23,20 @@ def httpbin_secure_untrusted(monkeypatch, httpbin_secure):
     """
     monkeypatch.delenv('REQUESTS_CA_BUNDLE')
     return httpbin_secure
+
+
+@pytest.fixture(autouse=True, scope='session')
+def pyopenssl_inject():
+    """
+    Injects `pyOpenSSL` module to make sure `requests`
+    will use it.
+
+    <https://github.com/psf/requests/pull/5443#issuecomment-645740394>
+    """
+
+    if os.getenv('HTTPIE_TEST_WITH_OPENSSL', default='0') == '1':
+        try:
```

Comment:
Here I’d rather try to explicitly import PyOpenSSL directly and show an error that it needs to be installed. The injecting part itself should not be inside a try…catch.